Bento/Components/DetailedDescription.swift

```diff
@@ -0,0 +1,439 @@
+private let maxImageWidth: CGFloat = 100
+
+public extension Component {
+    /// `DetailedDescription` supports a horizontal layout comprising:
+    ///
+    /// (1) [Optional] an image or a text placeholder, at most 100 points in width,
+    ///     with an optional badge icon whose position is customisable;
+    /// (2) [Optional] a vertical stack of zero or more independent text labels;
+    /// (3) [Optional] a single-line detail label; and
+    /// (4) [Optional] an accessory.
+    ///
+    /// While `DetailedDescription` supports asynchronous loading for its image
+    /// view, you are obligated to ensure a consistent fixed size across all
+    /// changes.
+    public final class DetailedDescription: AutoRenderable, Focusable {
```

Comment:
this is `TitledDescription` but without Property?

Reply:
Yes, it's exactly the same, except that the image property is not a reactive `Property`.